### PR TITLE
correct font

### PR DESCRIPTION
**Pull request by hmoldenhauer**

### CODE REVIEW

File: documents/satzung/gebuehrenordnung.tex [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
+	DIV14,
+	parskip,
+]{scrreprt}
+
+\usepackage{fontspec}
+\setmainfont{Source Sans Pro}
+\renewcommand\familydefault\sfdefault
 
 \usepackage[ngerman]{babel}
 \usepackage[utf8]{inputenc}
```
Comment (maxnoe) [inline documents/satzung/gebuehrenordnung.tex]: When using fontspec / lualatex, eurosym, inputenc and fontenc should not be used.

Comment (hmoldenhauer) [reply documents/satzung/gebuehrenordnung.tex]: Do you have a quick fix?

Comment (maxnoe) [reply documents/satzung/gebuehrenordnung.tex]: just remove

Comment (hmoldenhauer) [reply documents/satzung/gebuehrenordnung.tex]: removed

**Review by maxnoe — APPROVED** [review]

**Pull request merged**